src/EntityManagerFactory.php

```diff
@@ -19,17 +18,10 @@
 class EntityManagerFactory extends AbstractFactory
 {
     /**
-     * @var bool
-     */
-    private static $areTypesRegistered = false;
```

Comment:
well remowing this could be BC break but don't think someone is using this somehow :)